RetailCoder.VBE/UI/Inspections/InspectionResultsViewModel.cs

```diff
@@ -486,21 +510,9 @@ private bool CanExecuteCopyResultsCommand(object parameter)
             return !IsBusy && _results != null && _results.Any();
         }
 
-        public Visibility EmptyUIRefreshVisibility
-        {
-            get
-            {
-                return _state.Projects.Count > 0 ? Visibility.Hidden : Visibility.Visible;
-            }
-        }
+        public Visibility EmptyUIRefreshVisibility => _state.Projects.Count > 0 ? Visibility.Hidden : Visibility.Visible;
 
-        public Visibility EmptyUIRefreshMessageVisibility
-        {
-            get
-            {
-                return _isBusy ? Visibility.Hidden : Visibility.Visible;
-            }
-        }
+        public Visibility EmptyUIRefreshMessageVisibility => _isBusy ? Visibility.Hidden : Visibility.Visible;
```

Comment:
should access the Property to be consistent with line 510

Reply:
So converting both back to the way it was?

Reply:
no. this would be `=> IsBusy ?  [...]`